Daliana Driga - Tema 1

DalianaDriga opened the pull request:
Tema 1 done; am scris 9 test cases.

istvan19 committed:
tema1

DalianaDriga committed:
Tema 1 Daliana Driga

DalianaDriga committed:
Tema 1 Daliana Driga: small changes.

andreienea1990 commented on src/test/resources/FeatureFiles/login.feature:
```diff
@@ -0,0 +1,63 @@
+
+  Scenario Outline: Log in with valid username and password
+    When I enter a valid username and password
+    And I click Login
```
nu se foloseste corect username password cauta sa vezi cum trimiti variabilele dintr-un tabel de examples

DalianaDriga replied:
Am schimbat cu <userName> si <password> si am formatat tabelul.

andreienea1990 commented on src/test/resources/FeatureFiles/login.feature:
```diff
@@ -0,0 +1,63 @@
+    Then I am able to log in
+
+  Scenario: Log in w/o credentials by using the 'Back' and 'Forward' buttons
+    Given I am logged in to https://www.saucedemo.com/
```
foloseste deja Background nu e nevoie

DalianaDriga replied:
Am sters Given

DalianaDriga committed:
Tema 1 Daliana Driga:
- am updatat exemplul din scenariul 'Log in w/o credentials by using the 'Back' and 'Forward' buttons'
- am sters Given din scenariul 'Log in with valid username and password'

DalianaDriga committed:
Tema 1 Daliana Driga: Small fix to 'Log in using 'Tab' key to navigate and 'Enter' to log in' si am adaugat When la 'Log in w/o credentials by using the 'Back' and 'Forward' buttons'

istvan19 committed:
selenium 101

DalianaDriga committed:
Merge remote-tracking branch 'origin/Istvan_Gyorfi_AQA_PRE_2022' into Daliana_Driga-_AQA_PRE_2022

# Conflicts:
#	pom.xml

DalianaDriga committed:
updates

DalianaDriga committed:
Tema 2 Daliana Driga: Ongoing - First test written

monklord19 commented on src/test/resources/FeatureFiles/login.feature:
```diff
@@ -1,62 +1,83 @@
-  Background:
+  @SuccessfulLogin
+  Scenario: Log in with valid username and password
     Given I open https://www.saucedemo.com/
```
Foarte bine facut. Felicitari.
Aici te-ai incurcat un pic cu Scenario Outline. Dar daca ai vreme maine putem sa ne uitam

monklord19 reviewed (commented):
Ugyes

DalianaDriga committed:
Tema 2 Daliana Driga: Fixed first test written

DalianaDriga committed:
Tema Daliana Driga: Spotify Login (ongoing)

DalianaDriga committed:
Tema Daliana Driga: Spotify (Am creat page objects pentru Homepage si Login page)

DalianaDriga committed:
Tema Daliana Driga: Postman Tema 1 (exported collection)

DalianaDriga committed:
Tema Daliana Driga: Postman API tests (reqres.in)

DalianaDriga committed:
Tema Daliana Driga: PostmanApiTest.java file pushed  (reqres.in)

DalianaDriga committed:
Tema Daliana Driga: Tested serialization/deserialization

DalianaDriga committed:
Daliana Driga: Changes to the Reqres stepdefs (mai precis, testele pe care le-am discutat in penultima sesiune de training)

DalianaDriga committed:
Daliana Driga: Reqres step defs updated (am adaugat 3 teste care functioneaza: 2 get si 1 post)

DalianaDriga committed:
Daliana Driga: Reqres step defs updated (am adaugat Put)
